cdci_data_analysis/analysis/parameters.py

```diff
@@ -745,14 +746,12 @@ def check_energy_value(value, units=None,name=None):
             raise RuntimeError('type of ',name,'not valid',type(value))
 
 
-
-
-
-class DetectionThreshold(Parameter):
+class DetectionThreshold(Float):
     def __init__(self,value=None,units='sigma',name=None):
 
         _allowed_units = ['sigma']
-
+        if units is None or units == '':
+            units = 'keV'
```

Comment:
`'sigma'`. Also let's set it to default units.

Reply:
indeed how I do now, inside Parameter directly